### ISSUE TITLE
notifications are still being pushed to uppers after they left a part-up

### ISSUE THREAD

**Issue by PhilH** [part-up/part-up#395]
Given I am a member of a part-up
When I leave the part-up
Then I want not to receive notifications from this part-up anymore

// tested with
// - philh@erisma.com user
// - "Google analytics instellen voor platform en blog" room

**Comment by ralphboeije** [comment]
@PhilH can you please show a screenshot and state the type of notification you receive?

**Comment by PhilH** [comment]
Here's the notification in the app:
![notif1](https://cloud.githubusercontent.com/assets/53337/14782823/efd729ea-0aeb-11e6-9474-41c7bd7c3e5d.png)

The email:
![notif2](https://cloud.githubusercontent.com/assets/53337/14782847/1734caf6-0aec-11e6-8a42-069135619486.png)

As shown here, i'm mentioned in the comments, but I'm no longer part of the part-up:
![notif3](https://cloud.githubusercontent.com/assets/53337/14782882/4eaeaec0-0aec-11e6-9565-b513204f573e.png)

**Comment by ralphboeije** [comment]
Ah, check, this is because you were part of this conversation before. I agree that we shouldn't do this.

**Comment by PhilH** [comment]
Yes. It means that there's no way to stop receiving those notifications and escape spam :)
